internal/chain/engine.go

```diff
@@ -188,6 +188,11 @@ func (e *engineImpl) VerifySeal(chain engine.ChainReader, header *block.Header)
 	}
 	parentHash := header.ParentHash()
 	parentHeader := chain.GetHeader(parentHash, header.Number().Uint64()-1)
+	if parentHeader == nil {
+		return errors.New(
+			"[VerifySeal] no parent header found",
```

Comment:
might as well make it actually useful if we're doing this, `errors.WithStack(errors.Errorfparent for %s at height %d, ))`

Reply:
ack. can do next PR.